roles/mesos/defaults/main.yml

```diff
@@ -8,7 +8,8 @@ mesos_agent_package: "mesos-agent-{{ mesos_version }}"
 mesos_mode: follower
 mesos_log_dir: /var/log/mesos
 mesos_logging_level: WARNING
-mesos_work_dir: /var/lib/mesos
+mesos_work_dir: /mnt/mesos
+mesos_lvm_data_volume_size: 20%FREE
```

Comment:
@ryane Looks like sensible defaults to me, then :-) I think folks will just have to customize these for themselves. Can they be changed after the initial installation?